src/components/Auth/UserFormDialog/controls/MultiFactorAuth.tsx

```diff
@@ -116,7 +123,7 @@ export const MultiFactor: React.FC<
           </Typography>
           <div>
             {fields.map((item, index) => {
-              const fieldName = `mfaPhoneInfo[${index}].phoneInfo`;
+              const fieldName = `mfaPhoneInfo.${index}.phoneInfo` as const; 
```

Comment:
No need for `as const` on a string?

Reply:
FWIW, `as const` narrows the literal type to even more specific than `string`. The actual type is something more like `mfaPhoneInfo.${number}.phoneInfo` (i.e. TypeScript knows that it is a string with the format of a number between two fixed parts), which is needed for the library.